frontend/src/components/FeedEntryList.tsx

```diff
@@ -1,17 +1,32 @@
 import * as React from 'react';
-import { FeedEntry } from '../Connection';
+import Connection, { FeedEntry } from '../Connection';
 import FeedEntryView from './FeedEntry';
 
-export type FeedEntryListProps = {
+export type FeedEntryListPropState = {
   entries: FeedEntry[],
 }
 
+export type FeedEntryListPropDispatch = {
+  markRead: (entryId: number, conn: Connection) => void,
+}
+
+export type FeedEntryListProps = FeedEntryListPropState & FeedEntryListPropDispatch & { conn: Connection };
+
 class FeedEntryList extends React.Component<FeedEntryListProps> {
+  constructor(props: FeedEntryListProps) {
+    super(props);
+    this.markRead = this.markRead.bind(this);
```

Comment:
Should we be using arrow notation for these? Then we don't need the constructor.

Reply:
We should, now that I see it! I didn't know you could use arrow notation in classes like that :o